pkg/cmd/roachprod/flags.go

```diff
@@ -423,6 +423,10 @@ func initFlags() {
 		cmd.Flags().BoolVar(&insecure,
 			"insecure", insecure, "use an insecure cluster")
 	}
+
+	updateTargetsCmd.Flags().BoolVar(&insecure,
```

Comment:
I think we should add `updateTargetsCmd` to [this list](https://github.com/cockroachdb/cockroach/blob/b185402d34768c898d63a61de3e585e4723fbe81/pkg/cmd/roachprod/flags.go#L418).